chart/prometheus-msteams/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "v1.4.1"
 description: A Helm chart for Kubernetes
 name: prometheus-msteams
 home: https://github.com/prometheus-msteams/prometheus-msteams
-version: 0.7.1
+version: 0.8.0
```

Comment:
see my comment from previous review: this would be version `1.0.0` when we introduce a breaking change.